Formula/convox.rb

```diff
@@ -0,0 +1,15 @@
+class Convox < Formula
+  desc "The convox AWS PaaS CLI tool"
+  homepage "https://convox.com/"
+  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
```

Comment:
@mwarkentin Apologies, it takes me a couple of days to swing back round to a PR a lot of the time, fingers in too many pies, etc. Thanks for your efforts here, nice work 👍.